docs/UserGuide.md

```diff
@@ -153,6 +153,23 @@ Examples:
 * `find alex david` returns `Alex Yeoh`, `David Li`<br>
   ![result for 'find alex david'](images/findAlexDavidResult.png)
 
+### Locating persons by debt description: `finddebt`
+
+Finds persons who are associated with any debts that match any of the given keywords.
+
+Format: `finddebt <keyword> [<more keywords>]`
+
+* The search is case-insensitive. e.g `burger` will match `Burger`
+* The order of the keywords does not matter. e.g. `Sharing Meal` will match `Meal Sharing`
+* Only the name is searched.
+* Only full words will be matched e.g. `Burger` will not match `Burgers`
+* Persons with debts matching at least one keyword will be returned (i.e. `OR` search).
+  e.g. `burger meal` will return people associated with debts that have descriptions `Chicken Burger` or `Meal Sharing`
```

Comment:
I think these aren't needed since there are the same Notes above? Unless there are some new notes unique to `finddebt`.

Reply:
I think notes for each command should be mentioned explicitly, since some users may not look at the notes for other commands while searching for the command they need.

Reply:
@WR3nd3 @TheSoggy @lyuting47 what do yall think cos we do have to standardise across our user guide